modules/prometheus-exporter_oracle/conf/readme.yaml

```diff
@@ -0,0 +1,2 @@
+documentations:
+source_doc:
```

Comment:
this module will not be usable while there is no doc here to explain how to deploy the metrics collection

Reply:
as this is related to CLARANET (currently) and for my team SOS DBA, I have not plan to doc this module for public access, and the deployment is related only for claranet and particular ecosystem that I have on SOS DBA (standards etc) . For example custom metrics are managed in SOS DBA tooling (onegit) and are related to our standards, and are not usable by others company.

=> maybe we could discuss quickly about this global subject (documentation), to provide me some explanations in order to be "compliant" from your side :)

Reply:
feel free to join me on slack